cassandra-diagnostics-core/src/test/java/io/smartcat/cassandra/diagnostics/module/heartbeat/HeartbeatConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+package io.smartcat.cassandra.diagnostics.module.heartbeat;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.yaml.snakeyaml.constructor.ConstructorException;
+
+public class HeartbeatConfigurationTest {
+
+    @Test
+    public void testDefaultValues() throws Exception {
```

Comment:
Added test naming guidelines [here](https://github.com/smartcat-labs/smartcat-guidelines/blob/master/coding-guidelines/README.md). Check it out and use it.